app/src/renderer/components/staking/PageDelegate.vue

```diff
@@ -1,37 +1,33 @@
 <template lang="pug">
-page(:title="pageTitle")
+page.page-delegate(title="Delegate Atoms")
   div(slot="menu"): tool-bar
     router-link(to='/staking')
       i.material-icons arrow_back
       .label Change Candidates
```

Comment:
i think back arrows should always say "back"

Reply:
Agreed.